[2025-04-21T00:01:51.167+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: edu_pipeline_etl.check_snowflake_connection manual__2025-04-21T00:01:46.384508+00:00 [queued]>
[2025-04-21T00:01:51.176+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: edu_pipeline_etl.check_snowflake_connection manual__2025-04-21T00:01:46.384508+00:00 [queued]>
[2025-04-21T00:01:51.178+0000] {taskinstance.py:2170} INFO - Starting attempt 1 of 1
[2025-04-21T00:01:51.194+0000] {taskinstance.py:2191} INFO - Executing <Task(PythonOperator): check_snowflake_connection> on 2025-04-21 00:01:46.384508+00:00
[2025-04-21T00:01:51.202+0000] {standard_task_runner.py:60} INFO - Started process 135 to run task
[2025-04-21T00:01:51.205+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'edu_pipeline_etl', 'check_snowflake_connection', 'manual__2025-04-21T00:01:46.384508+00:00', '--job-id', '4', '--raw', '--subdir', 'DAGS_FOLDER/edu_pipeline_etl.py', '--cfg-path', '/tmp/tmpsksqtflj']
[2025-04-21T00:01:51.209+0000] {standard_task_runner.py:88} INFO - Job 4: Subtask check_snowflake_connection
[2025-04-21T00:01:51.278+0000] {task_command.py:423} INFO - Running <TaskInstance: edu_pipeline_etl.check_snowflake_connection manual__2025-04-21T00:01:46.384508+00:00 [running]> on host 5edb91fd105d
[2025-04-21T00:01:51.373+0000] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='edu_pipeline_etl' AIRFLOW_CTX_TASK_ID='check_snowflake_connection' AIRFLOW_CTX_EXECUTION_DATE='2025-04-21T00:01:46.384508+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-04-21T00:01:46.384508+00:00'
[2025-04-21T00:01:51.375+0000] {connection.py:370} INFO - Snowflake Connector for Python Version: 3.6.0, Python Version: 3.9.18, Platform: Linux-5.15.167.4-microsoft-standard-WSL2-x86_64-with-glibc2.36
[2025-04-21T00:01:51.378+0000] {connection.py:1171} INFO - This connection is in OCSP Fail Open Mode. TLS Certificates would be checked for validity and revocation status. Any other Certificate Revocation related exceptions or OCSP Responder failures would be disregarded in favor of connectivity.
[2025-04-21T00:01:52.656+0000] {cursor.py:1028} INFO - query: [SELECT CURRENT_USER(), CURRENT_VERSION()]
[2025-04-21T00:01:52.800+0000] {cursor.py:1041} INFO - query execution done
[2025-04-21T00:01:52.803+0000] {cursor.py:1205} INFO - Number of results in first chunk: 1
[2025-04-21T00:01:52.812+0000] {logging_mixin.py:188} INFO - ✅ Connexion Snowflake OK — utilisateur : ADERDER, version : 9.9.2
[2025-04-21T00:01:52.816+0000] {connection.py:718} INFO - closed
[2025-04-21T00:01:52.893+0000] {connection.py:724} INFO - No async queries seem to be running, deleting session
[2025-04-21T00:01:52.970+0000] {python.py:201} INFO - Done. Returned value was: None
[2025-04-21T00:01:53.014+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=edu_pipeline_etl, task_id=check_snowflake_connection, execution_date=20250421T000146, start_date=20250421T000151, end_date=20250421T000153
[2025-04-21T00:01:53.111+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2025-04-21T00:01:53.152+0000] {taskinstance.py:3280} INFO - 0 downstream tasks scheduled from follow-on schedule check
